### ISSUE TITLE
Flashing CC1110 using Raspberry pi over debug pins (not uart)

### ISSUE THREAD

**Issue by viveksoni95** [tobyjaffey/cctl#2]
Hi, 

I want to flash the cc1110 but I can't use the bootloader as I have a customized cc1110 hardware and uart ports are not exposed.  
Is there any way to flash the cc1110 using raspberry pi over debug pins (not uart)?

Thank you

**Comment by ringtailsoftware** [comment]
Yes, use ccpil https://github.com/tobyjaffey/cctl/tree/master/ccpil

**Comment by viveksoni95** [comment]
Hi, 
I tried ccpil but I am having some issues. 
Attaching the screenshot of the issue.
![ccpil_issue](https://user-images.githubusercontent.com/4894840/46468441-b9da0c00-c7d0-11e8-9416-be4c4d29cb94.png)
